kitsune/products/api.py

```diff
@@ -115,7 +111,7 @@ class TopicList(LocaleNegotiationMixin, generics.ListAPIView):
     serializer_class = RootTopicSerializer
 
     def get_queryset(self):
-        queryset = self.queryset.filter(product__slug=self.kwargs["product"])
+        queryset = self.queryset.filter(products__slug=self.kwargs["product"], is_archived=False)
```

Comment:
Nit. I don't think the `is_archived=False` is needed because we're using the `active` manager above when defining the `queryset`?